R/checks.R

```diff
@@ -17,26 +21,38 @@ get_template_wildcards = function(parent_key,template_key){
   return(unlist(wildcards))
 }
 
-copy_no_clobber = function(from_file,to_file,force=FALSE){
+
+copy_no_clobber = function(from_file,
+                           to_file,
+                           force = FALSE){
+
   to_dir = dirname(to_file)
   suppressMessages(suppressWarnings(dir.create(to_dir,recursive = T)))
   print(paste("COPYING",from_file,"TO",to_file))
   if(force){
     file.copy(from_file,to_file)
   }
 }
-#' Check for a remote session and automagically confirm setup will work properly
+
+
+#' @title Check Remote Configuration.
+#' 
+#' @description Check for a remote session and automagically confirm setup will work properly.
 #' 
-#' This function determines if a user is working in GAMBLR remotely and, if so, will
+#' @details This function determines if a user is working in GAMBLR remotely and, if so, will
 #' check if their config is loaded properly and ssh_session is available. 
 #'
 #' @param auto_connect Set to TRUE to ensure an ssh_session is created if absent
 #'
 #' @return
```

Comment:
Does it return anything? I think maybe a boolean TRUE/FALSE depending on whether there is a remote connection?

Reply:
Yes, I have updated the return description for this function (it actually returns the Boolean for `check_host`).